opentelemetry-ruby.gemspec

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'opentelemetry-ruby'
+  spec.version       = OpenTelemetry::VERSION
+  spec.authors       = ['Ian Quick', 'Francis Bogsyani']
```

Comment:
s/Bogsyani/Bogsanyi/

Reply:
I totally didn't look up your name and did it from memory.  :(